ui/src/icons/Iconography.mdx

```diff
@@ -0,0 +1,23 @@
+import { IconGallery, IconItem, Meta } from '@storybook/blocks';
+
+import * as Icons from '@/icons';
+
+<Meta title="Iconography" />
+
+# Iconography
+
+<IconGallery style={{display: 'grid', gridTemplateColumns: 'repeat(auto-fill, minmax(calc(33.33% - 10px), 1fr))', gap: '10px'}}>
+  {Object.keys(Icons).map(iconName => {
+    if (iconName.startsWith('Icon')) {
```

Comment:
**nitpick (non-blocking)**: I wonder if it might be better to use `Icon` as a suffix over a prefix. `IconEvent` sounds weird to me. It feels inverted. Would that cause any issue?

```suggestion
    if (iconName.endsWith('Icon')) {
```

Reply:
🤔 This is a good reflection. It was the pattern already in this repo, I guess it might have been because when you are importing the icons you don't have to remember their names? Just start typing Icon and the list comes. We can change it, but we also need to change the names inside each icon file